protected/components/StoredDatasetMainSection.php

```diff
@@ -73,10 +73,7 @@ public function getReleaseDetails(): array
         $release_details = [];
         $doi_prefix = Yii::app()->params['mds_prefix'];
 
-        $author_sql = "select a.id, a.surname, a.first_name, a.middle_name, a.custom_name from author a, dataset_author da, dataset d where a.id=da.author_id and d.id = da.dataset_id and d.id=:id order by rank ASC, a.surname ASC, a.first_name ASC, a.middle_name ASC";
-        $command = $this->_db->createCommand($author_sql);
-        $command->bindParam(":id", $this->_id, PDO::PARAM_INT);
-        $authors_result = $command->queryAll();
+        $authors_result = \GigaDB\models\Author::listByDatasetId($this->_id, $this->_db);
         if (!empty($authors_result)) {
```

Comment:
imho, this part could be removed and $release_details['authors'] could be constructed directly

Reply:
Thanks for the suggestion, will look into that.

Reply:
Refactored.